### PR TITLE
No need to escape '<' and '>' inside code block

### PR DESCRIPTION
**Pull request by jonathandotchin**

### CODE REVIEW

File: docs/standard/collections/commonly-used-collection-types.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Collections are based on the [ICollection](https://docs.microsoft.com/dotnet/cor
...
...t/core/api/System.Collections.Stack) class and the [Stack&lt;T&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.Stack-1) generic class provide last-in-first-out lists. The [SortedList](https://docs.microsoft.com/dotnet/core/api/System.Collections.SortedList) class and the [SortedList&lt;TKey, TValue&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.SortedList-2) generic class provide sorted versions of the [Hashtable](https://docs.microsoft.com/dotnet/core/api/System.Collections.Hashtable) class and the [Dictionary&lt;TKey, TValue&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.Generic.Dictionary-2) generic class. The elements of a `Hashtable` or a `Dictionary<TKey, TValue>` are accessible only by the key of the element, but the elements of a `SortedList` or a [KeyedCollection&lt;TKey, TItem&gt;](https://docs.microsoft.com/dotnet/core/api/System.Collections.ObjectModel.KeyedCollection-2) are accessible either by the key or by the index of the element. The indexes in all collections are zero-based, except [Array](https://docs.microsoft.com/dotnet/core/api/System.Array), which allows arrays that are not zero-based.
```
Comment (BillWagner) [inline docs/standard/collections/commonly-used-collection-types.md]: Good change.

I think we should use the code blocks where we have links, for consistency and simplicity. For example:

...and the [`Queue<T>`](https://...)

What do you think?

**Comment by BillWagner** [comment]
@jonathandotchin Thanks again for the contribution.

I wonder if we want to use the `` syntax even where it's a link target. What are your thoughts?

File: docs/standard/collections/commonly-used-collection-types.md (hunk 1 above)
Comment (jonathandotchin) [inline docs/standard/collections/commonly-used-collection-types.md]: That would certainly make sense. After all, it is code.

**Comment by mairaw** [comment]
I was going to open an issue today to convert these API links into the @ format. That way we don't even have to worry about the brackets.

**Comment by jonathandotchin** [comment]
image
The above screenshot shows how it looks like with changes. 
Couple of advantages I see: more readable, no worry about backets (as mairaw mentioned)

but it does raise one question. When should it be a link versus not a link. In the above screenshot, the 3 code blocks are in fact the same content except some are link or the others not.

**Comment by mairaw** [comment]
@jonathandotchin is this using the @ symbol? If so, I'm wondering if it just renders as a link the first time, and then subsequent uses as normal text. We could ask for a different behavior. 
MSDN renders all of them as links: https://msdn.microsoft.com/en-us/library/0ytkdh4s(v=vs.110).aspx

**Comment by jonathandotchin** [comment]
@mairaw 
No, I didn't use the @ symbol. This is how the code looks like.

`Collections are based on the [`\``ICollection`\``](https://docs.microsoft.com/dotnet/core/api/System.Collections.ICollection) interface, the [`\``IList`\``](https://docs.microsoft.com/dotnet/core/api/System.Collections.IList) interface, the [`\``IDictionary`\``](https://docs.microsoft.com/dotnet/core/api/System.Collections.IDictionary) interface, or their generic counterparts. The `\``IList`\`` interface and the `\``IDictionary`\`` interface are both derived from the `\``ICollection`\`` interface; therefore, all collections are based on the `\``ICollection`\`` interface either directly or indirectly.`

**Comment by mairaw** [comment]
Oh got it! This is the original document. @stevehoag any reasons why you styled the APIs differently on first mention vs. second mention?

**Comment by stevehoag** [comment]
@mairaw This was the first doc that we ported from MSDN, and we hadn't yet decided on a convention (link on first mention vs. link on every mention. At the time linking to members was broken, so I chose the former for simplicity. I'm okay with restyling them now.

**Comment by BillWagner** [comment]
@jonathandotchin Let's do this:

. Can you update this topic to use the `Queue<T>` style syntax in cases with links, and cases without. (The link color is enough of a clue for readers where links exist).

@Mairaw is going to write the issue that we will (in time) update all links to use the '@' syntax she mentioned. See this for an example: https://github.com/dotnet/core-docs/blob/master/api/index.md

We won't hold this up for those changes. So, once you make the update mentioned above, we'll merge this in. It is a big improvement in readability for authors.

**Comment by jonathandotchin** [comment]
@BillWagner 
I'm not 100% sure what you mean by

> use the  `Queue<T>`  style syntax in cases with links, and cases without

Do you mean removing all hyperlinks and simply use code block?

**Comment by BillWagner** [comment]
@jonathandotchin Sorry about not being clear.

I mean put all the type names inside back-ticks, whether they are links or not. That way, we don't need to escape the < and > characters.

the end result would be:

. Every type name is in the code style.
. Those instances of type names that are links would also have the link brackets as well as the code style.

**Comment by jonathandotchin** [comment]
@BillWagner Ah got it
I'll take care of it

**Comment by BillWagner** [comment]
closing in favor of #794

**Pull request closed without merging**